File: branches/Medusa/Source/Fantasy.Jobs/Scheduling/TemplateAction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Linq;
using System.Runtime.Serialization;
using Fantasy.XSerialization;

namespace Fantasy.Jobs.Scheduling
{
    [DataContract]
    [XSerializable("templateAction", NamespaceUri = Consts.ScheduleNamespaceURI)]
    public class TemplateAction : Action
    {

        public override ActionType Type
        {
            get { return ActionType.Template; }
        }

        [DataMember]
        [XAttribute("template")]
        public string Template { get; set; }
    }
}

File: AutoConverted/Fantasy.Jobs/Scheduling/TemplateAction.java
package Fantasy.Jobs.Scheduling;

import Fantasy.XSerialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[DataContract, XSerializable("templateAction", NamespaceUri = Consts.ScheduleNamespaceURI)]
public class TemplateAction extends Action
{

	@Override
	public ActionType getType()
	{
		return ActionType.Template;
	}

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[DataMember, XAttribute("template")]
	private String privateTemplate;
	public final String getTemplate()
	{
		return privateTemplate;
	}
	public final void setTemplate(String value)
	{
		privateTemplate = value;
	}
}